src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -577,21 +577,30 @@ const PublicationReleasePage: NextPage<Props> = ({ releaseVersion }) => {
   );
 };
 
-export const getServerSideProps: GetServerSideProps<Props> = withAxiosHandler(
-  async ({ query }) => {
+export const getServerSideProps: GetServerSideProps<Dictionary<unknown>> =
+  withAxiosHandler(async ({ query }) => {
     const { publication: publicationSlug, release: releaseSlug } =
       query as Dictionary<string>;
 
     const releaseVersion = await (releaseSlug
       ? publicationService.getPublicationRelease(publicationSlug, releaseSlug)
       : publicationService.getLatestPublicationRelease(publicationSlug));
 
+    // EES-5951 Redirect to latest release URL if the requested URL is for the publication base
```

Comment:
We can remove the comment, the code can speak for itself and git history can give any extra clarity